robottelo/entities.py

```diff
@@ -394,12 +392,13 @@ def create_missing(self, auth=None):
         * the ``template_kind`` instance attribute is unset.
 
         """
-        super(ConfigTemplate, self).create_missing(auth)
+        super(ConfigTemplate, self).create_missing()
         if (vars(self).get('snippet') is False and
                 'template_kind' not in vars(self)):
             # A server is pre-populated with exactly eight template kinds. We
             # use one of those instead of creating a new one on the fly.
             self.template_kind = TemplateKind(
+                self.server_config_,
```

Comment:
The reason why the server config is needed here and in other places is because this will allow using a different configuration even though the Robottelo have its default config, isn't it?